test/karma.ci.conf.js

```diff
@@ -17,8 +17,8 @@ module.exports = function(config) {
         browsers: ['Chrome_travis_ci', 'Firefox', 'PhantomJS'],
         logLevel: config.LOG_INFO,
         preprocessors: {
-            'components/**/*.js': ['coverage']
+            'src/js/**/!(*spec|*helpers).js': ['coverage']
```

Comment:
I seem to remember that we keep coverage on test files in edx-platform so that we can be sure that all the tests are being run. Also, the helpers are actually part of the public API for the UI Toolkit, so it would be good to keep coverage for them too (in fact, we might even want to have tests for them, if that makes sense).

Was it causing issues having them run through coverage?